tests/conftest.py

```diff
@@ -48,26 +48,36 @@ def tmp_obj():
     resp = client.put_object(test_container, path, body=body)
     assert resp.status_code == HTTPStatus.OK, 'create failed'
 
-    yield Obj(f'/{test_container}/{path}', body)
+    body = datetime.now().isoformat().encode('utf-8')
```

Comment:
- If we add more than one file, need to change the fixture name.
- Use a 'for' loop to create all the files